ssms/basic_simulators/simulator.py

```diff
@@ -492,6 +523,32 @@ def simulator(
 
     # Multi-particle models
 
+    #   LBA-based models 
+        
+    # lba_sd = 0.1
+    if model == "lba_3_v1":
```

Comment:
body of if statement for `lba_3_v1` and `lba_angle_3_v1` are the same, is that intended?

Reply:
yes, that is intended